docs/distros/ubuntu.md

```diff
@@ -30,6 +30,24 @@ As an addition, `tezos-baking` package provides `tezos-baking-<network>` service
 systemd units for `tezos-node` and `tezos-baker-<proto>`.
 Configuration files for these services are located in `/etc/default/tezos-baking-<network>`.
 
+<a name="debian"></a>
+## Ubuntu packages on Debian
+
+You can add the PPA using:
+```
+# Install software properties commons
+sudo apt-get install software-properties-common gnupg
+# Add PPA with Tezos binaries
+sudo add-apt-repository 'deb http://ppa.launchpad.net/serokell/tezos/ubuntu bionic main'
```

Comment:
The problem with Debian is that using `bionic` here only works on some Debian versions, but not others.

As mentioned in the issue description:
> (see [related askubuntu thread](https://askubuntu.com/a/445496)).

which needs to be used to figure out what to pick in this command.

Reply:
I checked on bullseye and buster, it works

Reply:
Ah, I see, nice to hear! 

AFAICT It looks like:
1. `stretch` is still under LTS, but we no longer support the equivalent Ubuntu
2. `bookworm` maybe we could support, but OTOH it's still in "testing" phase, so we don't need to right now

so, it looks like currently these two are all we can support.

Most likely we'll need to update this in the future, but right now these instructions LGTM :ok_hand: